### DISCUSSION TITLE
Latest Release and Linux Kernel

### DISCUSSION THREAD

**Discussion by jeffreyflynt** [edgelesssys/constellation#3261] [Q&A] [answered]
Hello,

I installed constellation on Azure and when I run: `cat /proc/cpuinfo` the avx/avx2 flags are not present. When viewing the console log for an instance, I see this: `mitigations=auto,nosmt`.

Does anyone know if `avx` has been disabled with the new kernel or Fedora 40 release? I use the same exact instance types but `avx` is somehow disabled with the upgrade.

**Comment by derpsteb** [comment, marked as answer]
Hey, 
thanks for using Constellation!

We can reproduce the behavior you are seeing and are currently investigating how to fix the issue. We are tracking this in issue #3262 . A workaround for this issue is to use v2.16.4.

Cheers.
